trying to fix testX

jerryzh168 opened the pull request:
Summary: passed locally but seems testX has some problem

Differential Revision: D13820250

ezyang reviewed (approved):
If you want to test device=1 case, you need to also assert two or more GPUs.

jerryzh168 commented:
testing device_id == 0 should be good enough

jerryzh168 committed:
trying to fix testX (#16370)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/16370

passed locally but seems testX has some problem

Reviewed By: ezyang

Differential Revision: D13820250

fbshipit-source-id: 2572e154c25d4a153eb7355237b1930359e36820
